en/middleware/xrce_dds.md

```diff
@@ -205,17 +217,21 @@ To use these ports you must first disable the existing configuration:
   See [Serial port configuration](../peripherals/serial_configuration.md#serial-port-configuration).
 :::
 
-Once set, you may need to reboot PX4 for the parameters to take effect. They will then perist through subseqent reboots.
-
+Once set, you may need to reboot PX4 for the parameters to take effect.
+They will then perist through subsequent reboots.
 
 While not recommended, you can also start the [microdds-client](../modules/modules_system.md#microdds-client) using a command line.
 This can be called as part of [System Startup](../concept/system_startup.md) or through the [MAVLink Shell](../debug/mavlink_shell.md) (or a system console).
-For example:
+For example, the following command can be used to connect via Ethernet to a remote host at `192.168.0.100:8888`.
 
 ```sh
-microdds_client start -t serial -d /dev/ttyS3 -b 921600
+microdds_client start -t udp -p 8888 -h 192.168.0.100
 ```
 
+:::note
+At time of writing there is no PX4 parameter for setting the address of the remote host on which the XRCE-DDS agent is running.
+Therefore for Ethernet connections you will _have_ to use a command to start the client.
+:::
```

Comment:
FYI @beniaminopozzan This "fixes" the docs to explain you need to use command line for Ethernet, as discussed in https://github.com/PX4/PX4-user_guide/pull/2259/commits/986995c6a4309cc496d17c67659a336ded449045#r1108235444

If you give me an issue for creating the host parameter for pointing to the agent I'll add that as a link from "at the time of writing" in the note.

Reply:
Here it is!
https://github.com/PX4/PX4-Autopilot/issues/21179

Reply:
Thanks!!! I'll add to docs.